packages/js-config-webpack-plugin/src/JsConfigWebpackPlugin.js

```diff
@@ -33,13 +33,25 @@ class JsConfigWebpackPlugin {
 	 * @returns {string}
 	 */
 	resolveBabelConfigFilePath(contextPath, environmentName) {
+		/**
+		 * Get the last result of an iterator
+		 * @param {Generator} iterator
+		 */
+		function getLast(iterator) {
+			let result = iterator.next();
+			while (!result.done) {
+				result = iterator.next();
+			}
+			return result.value;
+		}
+
```

Comment:
```suggestion
		/**
		 * Get the last result of an iterator
		 * @param {Generator | object} iterator
		 */
		function getLast(iterator) {
		    if (typeof iterator.next !== 'function') {
		        return iterator;
		    }
			let result = iterator.next();
			while (!result.done) {
				result = iterator.next();
			}
			return result.value;
		}

```

Reply:
I think this looks good 👍 

Maybe clarify that it returns the parameter if it's not an iterator?

Reply:
I've gone with something like this, however I've had to invert the condition. It is sometimes valid for `iterator` to be null, in which case `typeof iterator.next !== 'function'` fails.

Reply:
I've also updated the comments/identifiers to clarify a bit what is going on 👍